src/test/scala/uk/gov/hmrc/ct/ct600/v3/LoansToParticipatorsSpec.scala

```diff
@@ -83,7 +86,7 @@ class LoansToParticipatorsSpec extends WordSpec with Matchers {
         validLoan.copy(name = "Pneumonoultramicroscopicsilicovolcanoconiosisisgoodforme")))
 
       val errors = l2pBox.validate(boxRetriever)
-      errors.size shouldBe 0
+      errors shouldBe empty
```

Comment:
errors shouldBe empty  shows the reason of a failure in case of failure, whilst errors.size shouldBe 0 doesnt